examples/example-xds/README.md

```diff
@@ -0,0 +1,53 @@
+gRPC XDS Example
+================
+
+The XDS example is a Hello World client capable of being configured with the
+XDS management protocol. Out-of-the-box it behaves the same as hello world
+client.
+
+__XDS support is incomplete and experimental, with limited compatibility. It
+will be very hard to produce a working enviornment just by this example. Please
+refer to documentation specific for your XDS management server and
+environment.__
+
+The example requires grpc-java to already be built. You are strongly encouraged
+to check out a git release tag, since there will already be a build of grpc
+available. Otherwise you must follow [COMPILING](../../COMPILING.md).
+
+### Build the example
+
+1. Build the hello-world example server or the hostname example server. See
+   [the examples README](../README.md) or the
+   [hostname example README](../example-hostname/README.md).
+
+2. Build this client. From the `grpc-java/examples/examples-xds` directory:
+```
+$ ../gradlew installDist
+```
+
+This creates the script `build/install/example-xds/bin/xds-hello-world-client`
+that runs the example.
+
+To start the server, run:
+
+```
+$ ../build/install/hostname/bin/hello-world-server
+$ # or
+$ ../example-hostname/build/install/hostname/bin/hostname-server
+```
+
+And in a different terminal window run this client:
+
+```
+$ ./build/install/example-xds/bin/xds-hello-world-client
+```
+
+However, that didn't use XDS! To use XDS we assume you have deployed the server
+in your deployment environment and know its name. You need to set the
+`GRPC_XDS_BOOTSTRAP` environment variable to point to a gRPC XDS bootstrap
```

Comment:
There is no example of bootstrap. How about referencing https://github.com/grpc/proposal/pull/170 and in particular https://github.com/grpc/proposal/pull/170/files#diff-05ea4a5894abbc0261b006741220598cR100 when it's merged?